### DISCUSSION TITLE
[blog categories] Card Layout Rendering for the`Categories` Page

### DISCUSSION THREAD

**Discussion by ebouchut** [squidfunk/mkdocs-material#5918] [Ideas]
Today, the _blog_ plugin renders the **Categories page** with a list of all blog posts (title + excerpt) for the **selected** category.

I would like to propose an **alternative rendering** for this page, showing **all categories on a single page**, a kind of card layout (without an image).
This would provide a different way to browse all the existing categories.
This could be configured to show each category name as a section, with a subset of recent blog posts below (just titles this time, no excerpts).
The configuration parameters could be:
- `card_layout`: If enabled, display all categories using a 2-column layout.
- `post_per_category`: How many posts to show per category (most recent first)

<img width="1324" alt="CleanShot 2023-08-29 at 14 52 14@2x" src="https://github.com/squidfunk/mkdocs-material/assets/73331/d2b4db6b-a7ad-4d6f-8960-fe649feb1265">

[Source](https://www.justinjoyce.dev/)

**Comment by squidfunk** [comment]
Thanks for suggesting! That sounds like a job for [the tags plugin](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#built-in-tags-plugin), because it provides this exact functionality. I don't think it make sense to also implement this into the blog plugin, because it's already possible by using the tags plugin. Just switch `categories` to `tags` in your post. Related:

- #5913

**Reply by ebouchut** [reply]
Thank you @squidfunk, I will use the `tags` plugin.
